### ISSUE TITLE
Hadouken status

### ISSUE THREAD

**Issue by thecubical** [Sonarr/Sonarr#1572]
Looking at the [Hadouken statuses](https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.Core/Download/Clients/Hadouken/Models/HadoukenTorrentState.cs)

```csharp
public enum HadoukenTorrentState
{
    Unknown = 0,
    QueuedForChecking = 1,
    CheckingFiles = 2,
    DownloadingMetadata = 3,
    Downloading = 4,
    Finished = 5,
    Seeding = 6,
    Allocating = 7,
    CheckingResumeData = 8,
    Paused = 9
}
```
It seems they don't fully line up with what is being reported via the [JSON RPC](https://github.com/hadouken/hadouken/blob/develop/js/rpc/webui_list.js)

```javascript
var state = 0;
if(!status.isPaused) { state = (state | 1); } // started
if(status.state === 0
        ||status.state === 1
        || status.state === 7) { state = (state | 2); } // checking
if(status.error) { state = (state | 16); } // error
if(!status.autoManaged && status.isPaused) { state = (state | 32); } // paused
if(status.autoManaged && status.isPaused) { state = (state | 64); } // queued
```

Hadouken uses libtorrent which has the following [states](http://www.rasterbar.com/products/libtorrent/manual.html#torrent-status)

- queued_for_checking = 0
- checking_files =1
- downloading_metadata =2
- downloading =3
- finished =4
- seeding =5
- allocating = 6
- checking_resume_data = 7

It seems like 32 and 64 should be the values that are used in Sonarr 

Hitting the JSON RPC with a single torrent that is auto managed and paused returns the following:
![Image](http://i.imgur.com/UhlpA9X.png)
```javascript
{
  "id": 1,
  "jsonrpc": "2.0",
  "result": {
    "label": [
      [
        "tv",
        1
      ]
    ],
    "torrents": [
      [
        "xxxxxxxx",
        32,
        "atorrent",
        450437637,
        1000,
        454744599,
        0,
        0,
        0,
        0,
        -1,
        "tv",
        0,
        41,
        0,
        0,
        -1,
        null,
        0,
        "",
        "",
        "",
        "",
        1480835417,
        1480836697,
        "",
        "D:/Completed/"
      ]
    ]
  }
}
```
